### PR TITLE
Created_footer_and_header

### PR DESCRIPTION
**Pull request by azadehroshan**
#6 -Created_footer_and_header by Azadeh

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -2,43 +2,53 @@ import { useEffect, useState } from "react";
 import { Link } from "react-router-dom";
```
Comment (AnoshMalik) [inline client/src/pages/Home.js:2]: Remove this line please

File: client/src/Components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
Comment (AnoshMalik) [inline client/src/Components/Footer.js:1]: Use double quotes around 'react' please

File: client/src/pages/Home.js (hunk 1 above)
Comment (azadehroshan) [reply client/src/pages/Home.js:2]: done

**Review by AnoshMalik — CHANGES_REQUESTED** [review]
See individual messages for improvements

File: client/src/Components/Footer.js (hunk 2 above)
Comment (azadehroshan) [reply client/src/Components/Footer.js:1]: thanks-done

File: client/src/pages/Home.js (hunk 1 above)
Comment (Roman-Hal) [inline client/src/pages/Home.js:2]: Maybe to comment out this lines would avoid the conflict.

File: client/src/pages/Home.js [hunk 3]
```diff
@@ -2,43 +2,53 @@ import { useEffect, useState } from "react";
...
```
Comment (Roman-Hal) [inline client/src/pages/Home.js:3]: Commenting out the line might help to avoid the conflict.

**Review by Roman-Hal — CHANGES_REQUESTED** [review]
There is some problems showing by the system to be corrected.

File: client/src/pages/Home.js [hunk 4]
```diff
@@ -2,43 +2,53 @@ import { useEffect, useState } from "react";
...
 
 import "./Home.css";
-import logo from "./logo.svg";
+import Header from "../Components/Header";
+import Footer from "../Components/Footer";
+// import logo from "./logo.svg";
 
 export function Home() {
-	const [message, setMessage] = useState("Loading...");
```
Comment (youweb3) [inline client/src/pages/Home.js:8]: avoid this line

File: client/src/pages/Home.js [hunk 5]
```diff
@@ -41,14 +41,13 @@ export function Home() {
...
-);
-
+	return (
+		<div>
+			<Header />
+			<h1>Suggestion Generator</h1>
+			<Footer />
+		</div>
+	);
 }
```
Comment (youweb3) [inline client/src/pages/Home.js:51]: add semicolon in line 51

**Review by youweb3 — APPROVED** [review]

**Pull request closed without merging**